tests/openwrt/test_encryption.py

```diff
@@ -20,6 +112,7 @@ class TestEncryption(unittest.TestCase, _TabsMixin):
                         "protocol": "wpa2_personal",
                         "cipher": "tkip+ccmp",
                         "key": "passphrase012345",
+                        "ieee80211w": "2",
```

Comment:
Can you please move this  to a test that deals with WPA3? It will make more sense there.

Reply:
I would like to leave it because to confirm WPA2 can use ieee80211w=2.
Instead I modified existing WPA3 test to use ieee80211w=2.